apps/checker/test/scala/matchers/LanguageToolMatcherTest.scala

```diff
@@ -218,7 +218,7 @@ class LanguageToolMatcherTest extends AsyncFlatSpec with Matchers {
     val eventuallyMatches = instance.check(request)
 
     val expectedMatchMessages =
-      List("Did you mean <suggestion>fewer</suggestion>? The noun tests is countable.")
+      List("Did you mean <suggestion>fewer</suggestion>? The noun \"tests\" is countable.")
```

Comment:
I’m surprised this match text requires quotes but the one below (`The noun mistakes…`) doesn’t. What’s the difference?